tests/integration/seed/seedsVerification.test.js

```diff
@@ -200,7 +205,7 @@ describe('SEEDS VERIFICATION', () => {
           const someAccessRulesAreInDuplicate = courseList
             .filter(c => c.accessRules.length)
             .some((course) => {
-              const accessRulesWithoutDuplicates = [...new Set(course.accessRules.map(c => c.toHexString()))];
+              const accessRulesWithoutDuplicates = [...new Set(course.accessRules.map(c => c._id.toHexString()))];
```

Comment:
Comme tu renvoie une string si l'entreprise existe pas, on risque pas d'avoir une erreur plutot q'un test qui pete ici dans le cas ou on a un `new ObjectId()` ?

Reply:
si malheureusement... mais je sais pas trop comment résoudre ça. Je me demandais d'ailleurs si les tests existent étaient pertinent quand y avait un test duplicate car duplicates pétait déjà si on passait new ObjectId (c'est le cas aussi pour trainees et companies)

Reply:
Je ne pense pas que ce soit une bonne pratique d'avoir des tests qui renvoie des erreurs plutot que de faire en sorte qu'ils ne passent pas. 
Que penses-tu d'avoir un seul test "all companies in accessRules should exist and be present only once", qui vérifierait d'abord que toutes les entreprises existent, et si c'est le cas qui verifierait qu'effectivement elles sont toutes presente qu'une fois ?